src/lib/analytics.js

```diff
@@ -0,0 +1,43 @@
+// TODO: Add license info?
+
+import {CleanInsights} from 'clean-insights-sdk';
+import {matchPattern2RegEx} from './util';
+import mvelo from './lib-mvelo';
+import defaults from '../res/defaults.json';
+
+export const ci = new CleanInsights({
+  'server': 'https://metrics.cleaninsights.org/cleaninsights.php',
+  'siteId': 22,  // Mailvelope's site ID on Clean Insights Matomo instance.
+  'persistEveryNTimes': 1,
+  'debug': true,
+  'campaigns': {
+    'provider': {
+      'start': '2022-01-01T00:00:00-00:00',
+      'end': '2022-12-31T23:59:59-00:00',
+      'aggregationPeriodLength': 1, // days
+      'numberOfPeriods': 365,
+      // Record which sites are used each day.  Subsequent same-day visits aren't interesting.
+      'onlyRecordOnce': true,
+    },
+  }
+});
+
+// TODO: Actually seek consent
+// Manufacture consent during development
+ci.grantCampaign('provider');
+
+/* Record a hit to a domain in the watchList (i.e. where the mvelo content script was injected).
+ *
+ * To preserve privacy, we record only the matching site & frame and discard the remaining
+ * information in the URL.
+ */
+export function measureWatchListHit(url) {
+  defaults.watch_list.forEach(site => {
```

Comment:
When you use the defaults here, we will miss domains that the user has added manually to the watchlist. Was your intention to preserve privacy? I was also thinking in this direction that this can be seen as sensitive information.

Reply:
Exactly.  Open to other options, but this was my best guess for now.  Let's talk more Monday.